utils/sqlcommenter/test/Spec.hs

```diff
@@ -0,0 +1,52 @@
+{-# LANGUAGE OverloadedStrings #-}
+import Data.Text (Text)
+import Data.Foldable
+import Data.Maybe
+import qualified Data.Map.Strict as M
+import qualified Data.Text.Encoding as T
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+import Network.HTTP.Types (urlDecode)
+import OpenTelemetry.Trace.Core
+import Test.Hspec
+import Test.Hspec.Hedgehog
+import SqlCommenter
+
+textValGen :: Gen Text
+textValGen = Gen.text (Range.linear 1 400) Gen.unicode
+
+main :: IO ()
+main = hspec $ do
+  specify "Does not add comment if the query already has a comment" $ do
+    let queries =
+          [ "SELECT * FROM table -- noodle"
+          , "SELECT * FROM table -- noodle\n"
+          , "SELECT * FROM table /* noodle\n  poodle\n*/"
+          ]
+        someAttrs = M.fromList [("foo", "bar")]
+    for_ queries $ \query -> do
+      sqlCommenter query someAttrs `shouldBe` query
```

Comment:
I don't quite follow what this is testing; `someAttrs` is just ignored entirely and the query is returned as-is because there is an existing comment?

Reply:
Yes, exactly.

Per the sqlcommenter spec:

> If a comment already exists within a SQL statement, we MUST NOT mutate that statement.